core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/aggregator/AggregationType.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * 
+ */
+package edu.uci.ics.texera.dataflow.aggregator;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * @author avinash
+ *
+ */
+public enum AggregationType {
```

Comment:
Check other enum classes to see how to give them a lowercase name when converting this to JSON. 

`NlpEntityType.java` is a good example

Reply:
Done. But why exactly are lowercase names required?

Reply:
This comment hasn't been addressed yet, please check `NlpEntityType.java` https://github.com/Texera/texera/blob/master/core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/nlp/entity/NlpEntityType.java 

It's mainly for readability, these JSON values will be shown to users in the dropdown selection menu. lowercase english is better than uppercase in this scenario.